src/main/java/dev/snowdrop/release/services/JiraClientConfiguration.java

```diff
@@ -28,6 +29,7 @@
 @ApplicationScoped
 public class JiraClientConfiguration {
     @Produces
+    @DefaultBean
```

Comment:
I think that finally, since we are using the Quarkus Testing Profiles, we don't need mark this bean as `DefaultBean`. Quarkus will be able to use this bean when the `user` and `password ` will be passed as command params.

Reply:
I tried removing the `@DefaultBean` and got a `Ambiguous dependencies for type com.atlassian.jira.rest.client.api.JiraRestClient` so I'm keeping that.